content/en-us/reference/engine/classes/Status.yaml

```diff
@@ -19,8 +19,7 @@ deprecation_message: |
   work.
 
   Developers looking to implement custom `Class.Humanoid` statuses should use
-  `Class.CollectionService` which provides a more flexible and robust system for
-  tagging objects.
+  [Character Physics Controllers](https://devforum.roblox.com/t/releasing-character-physics-controllers/2623426), as they are written in Luau and are easily extendable.
```

Comment:
```suggestion
  [Character Physics Controllers](../../../physics/character-controllers.md), as they are written in Luau and are easily extendable.
```

Reply:
Changing link to the guide on this, which is the same feature as the forum post:
https://create.roblox.com/docs/physics/character-controllers

Reply:
I couldn't find that page while doing research, I'll change out the DevForum link for that!